db/mysql.js

```diff
@@ -13,14 +13,20 @@ hooker.hook(require('mysql/lib/Connection').prototype, 'query', {
 	}
 });
 
-const connection = mysql.createConnection({
-	host: 'localhost',
-	user: 'root'
-});
+let connection;
+if (process.env.MYSQL_URI) {
+	connection = mysql.createConnection(process.env.MYSQL_URI);
+} else {
+	connection = mysql.createConnection({
+		host: process.env.MYSQL_HOST || 'localhost',
+		user: process.env.MYSQL_USER || 'root'
+	});
+}
 
 // pretend we already connected
 connection._connectCalled = true;
 
+// TODO: rename this function?
 exports.register = function mongo(server, options) {
```

Comment:
Should this function be named `mongo`?

Reply:
yep, copypasta fail.

Reply:
yea you can rename

Reply:
also this repo is wonky. when you merge this make sure you merge to `master`, `hapi18` and `hapi17` they are 3 diff targets. master is latest hapi, hapi18 is obv 18 and so on